nixos/services/mail/default.nix

```diff
@@ -156,7 +156,7 @@ in {
         certificateScheme = 3;
         enableImapSsl = true;
         enableManageSieve = true;
-        lmtpSaveToDetailMailbox = "no";
+        # lmtpSaveToDetailMailbox = "no";
```

Comment:
can we just remove the line?

Reply:
Option is removed by one of the nixos-mailserver fc commits, should we drop those we'd need to re-enable it, I think

But need to check again

Reply:
Ah so option wasn't removed, this was just back when I pulled some random revision and had things break. We can leave it set to no.